app/views/posts/index.html.erb

```diff
@@ -15,7 +15,7 @@
 
     <div class='user-posts'>
         <h2>Posts</h2>
-        <% @posts.each do |post| %>
+        <% @posts.includes(:comments).each do |post| %>
```

Comment:
- [ ] Dear Team, you did a great job using the `include` keyword to preload the `comments` resource, but, please remember, as a common practice we need to include all the business/database-related logic inside a controller instead of a view. I suggest you did the same and implement this logic inside your `Posts_controller`.